templates/tfengine/recipes/cicd.hcl

```diff
@@ -53,6 +53,13 @@ schema = {
         type = "string"
       }
     }
+    managed_modules = {
+      description = "List of directories managed by the CICD relative to terraform_root."
```

Comment:
Also mention explicitly that the order defined in the list will be the order of deployment.

Reply:
Done